tasks/webhook

```diff
@@ -171,6 +173,19 @@ class ReleaseHandler(github_handler.GithubHandler):
         publish_to_queue('webhook', json.dumps(body))
         return None
 
+    def handle_issues_event(self, event, request):
+        action = request['action']
+        if event == 'issues' and action not in ['opened', 'synchronize', 'edited']:
+            logging.info("action not in ['opened', 'synchronize'], skipping issue")
```

Comment:
not the same set; how about making this more useful for debugging and say something like `"action %s unknown, skipping event" % action`?